File: src/test/resources/csharp/benchmark/Domain/Direct/Violating/CallConstructor_GenericType_MultipleTypeParameters.cs
using System.Collections.Generic;
using Technology.Direct.Dao;

namespace Domain.Direct.Violating
{
	public class CallConstructor_GenericType_MultipleTypeParameters
	{
		public CallConstructor_GenericType_MultipleTypeParameters()
		{
            dictionary = Dictionary<ProfileDAO, UserDAO>();
		}
	}
}

File: testresources/java/accuracy/src/domain/direct/violating/CallConstructor_GenericType_MultipleTypeParameters.java
package domain.direct.violating;

import java.util.HashMap;

import domain.direct.Base;
import technology.direct.dao.ProfileDAO;
import technology.direct.dao.UserDAO;

public class CallConstructor_GenericType_MultipleTypeParameters extends Base {
	public CallConstructor(){

		hashMap = new HashMap<ProfileDAO, UserDAO>();
	}
}